src/debug/debug-pal/CMakeLists.txt

```diff
@@ -25,6 +25,10 @@ if(CLR_CMAKE_PLATFORM_UNIX)
       unix/dynamiclibaddress.cpp
     )
 
+  if(CLR_CMAKE_PLATFORM_FREEBSD)
+    add_definitions(-D__FREEBSD__=1)
```

Comment:
Didn't know that. Pardon my cmake :)

I'll take that into account and tighten up the code accordingly.